app/config/services.yml

```diff
@@ -31,9 +31,15 @@ services:
         tags:
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest }
 
-    # Event subscribers are similar to event listeners but they don't need to add
-    # a separate tag for each listened event. Instead, the PHP class of the event
-    # subscriber includes a method that returns the list of listened events.
+    app.comment_notification:
+        class: AppBundle\EventListener\CommentNotificationListener
+        arguments: ['@mailer', '@router', '@translator', '%app.notifications.email_sender%']
+        tags:
+            - { name: kernel.event_listener, event: comment.created, method: onCommentCreated }
```

Comment:
I have two questions about the `onCommentCreated` method name:

1) When the method name is `on + camelCasedEventName`, the `method` parameter is optional. Should we add it anyway?
2) Some people like method names that describe the listened event (e.g. `on + EventName`) and others prefer semantic method names that describe their purposes (e.g. `notifyCommentAdded()`). Which is your preference?